helm/ffc-demo-payment-service/values.yaml

```diff
@@ -2,8 +2,12 @@ environment: development
 postgresUsername: postgres@mine-support2
 postgresPassword: changeme
 postgresDatabase: mine_payments
+# postgresHost is the host name of the PostgreSQL service
 postgresHost: ffc-demo-payments-postgres
-postgresExternalName: host.docker.internal
+# postgresExternalName is the external host name to which PostgreSQL
+# requests should be forwarded. If empty, PostgreSQL is assumed to be
+# within the cluster and accessible via postgresHost
+postgresExternalName:
```

Comment:
Now this value is empty in the default values file, do you think it's worth removing it from the `development-values.yaml` file also?  After this change it now overrides an empty value with another empty value.

Reply:
Yep, removed.